components/pages/workPageLayout/helper.ts

```diff
@@ -96,6 +69,24 @@ export const getManifestationLanguageIsoCode = (
   return undefined
 }
 
+export const materialTypeSortPriority: GeneralMaterialTypeCodeEnum[] = [
+  "BOOKS",
+  "EBOOKS",
+  "AUDIO_BOOKS",
+  "PODCASTS",
+  "BOARD_GAMES",
+  "ARTICLES",
+  "COMICS",
+  "COMPUTER_GAMES",
+  "FILMS",
+  "IMAGE_MATERIALS",
+  "MUSIC",
+  "NEWSPAPER_JOURNALS",
+  "OTHER",
+  "SHEET_MUSIC",
+  "TV_SERIES",
+]
```

Comment:
This, along with ..., should be moved to a config resolver. Like our [search config](https://github.com/danskernesdigitalebibliotek/dpl-go/blob/031bd8badc536f034d77931604224563aed1ef8f/lib/config/resolvers/search.ts#L41-L54).